src/hooks/useToastMessage.tsx

```diff
@@ -64,4 +66,16 @@ const useToastMessage = () => {
   return { showToast };
 };
 
+export const showToast = (title, error) => {
```

Comment:
Why is this new showToast?

Reply:
Ideally should use the same toast message, or at least rename it to showToastSimple or something, and make sure it is dismissed on press.